File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md

```diff
@@ -0,0 +1,267 @@
+control: DocIO
+documentation: UG
+---
+
+# Open and Save Word document in WinUI
+
+Syncfusion DocIO is a [WinUI Word library](https://www.syncfusion.com/document-processing/word-framework/winui/word-library) used to **create, read, and edit Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in WinUI**.
+
+**Prerequisites:**
+To use the WinUI 3 project templates, install the Windows App SDK extension for Visual Studio. For more details, refer [here](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=cs-vs-community%2Ccpp-vs-community%2Cvs-2022-17-1-a%2Cvs-2022-17-1-b).
+
+
+## WinUI Desktop app
+
+Step 1: Create a new C# WinUI Desktop app. Select Blank App, Packaged with WAP (WinUI 3 in Desktop) from the template and click the **Next** button.
+
+Step 2: Enter the project name and click **Create**.
+
+Step 3: Set the Target version to Windows 10, version 2004 (build 19041) and the Minimum version to Windows 10, version 1809 (build 17763) and then click **OK**.
+
+Step 4: Install the [Syncfusion.DocIO.NET](https://www.nuget.org/packages/Syncfusion.DocIO.Net) NuGet package as a reference to your project from the [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering a Syncfusion license key in your application to use our components.
+
+Step 5: Add a new button to the **MainWindow.xaml** as shown below.
+
+{% tabs %}
+
+{% highlight XML %}
+<Window
+    x:Class="CreateWordSample.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:CreateWordSample"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+        <Button x:Name="button" Click="CreateDocument">Create Document</Button>
+    </StackPanel>
+</Window>
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 6: Include the following namespaces in the **MainWindow.xaml.cs** file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 7: Add a new action method **OpenAndSaveDocument** in MainWindow.xaml.cs and include the below code snippet to **open an existing Word document in WinUI Desktop app**.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+private async void OnButtonClicked(object sender, RoutedEventArgs e)
```

Comment:
In sentence, method name mentioned as OpenAndSaveDocument, which is totally different in code example. Please modify the code example as 

  
Assembly assembly = typeof(Open_and_save_Word_document).GetTypeInfo().Assembly;
using (WordDocument document = new WordDocument(assembly.GetManifestResourceStream("Open_and_save_Word_document.Assets.Input.docx"), FormatType.Docx))